src/core/lib/iomgr/tcp_server_uv.c

```diff
@@ -76,13 +76,30 @@ struct grpc_tcp_server {
 
   /* shutdown callback */
   grpc_closure *shutdown_complete;
+
```

Comment:
I'm not sure that the first part is accurate. As far as I understand, it really is more of a "quota". You inform it when you need to allocate memory, and it tells you when that much of your quota is available.